keyboards/dotmod/dymium65/dymium65.h

```diff
@@ -0,0 +1,20 @@
+/* Copyright 2023 Finalkey
+ * Copyright 2023 LiWenLiu <https://github.com/LiuLiuQMK>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
```

Comment:
I intentionally removed this file on your previous PR. Why is it being added back?

Reply:
The person on the previous PR is quite a hardware guy and unfamiliar with git workflows...So every time he modifies a file, he deletes and reuploads it, which tends to override changes you required...
I am so sorry for that and I won't let it happen again.